backend/tests/v5/helper/fronteggMock/components/accounts.js

```diff
@@ -39,7 +39,12 @@ Accounts.createAccount = (name) => {
 	return Promise.resolve(accountId);
 };
 
-Accounts.getAllUsersInAccount = (accountId) => Promise.resolve(usersInAccount[accountId] ?? []);
+Accounts.getAllUsersInAccount = (accountId) => {
+	if (usersInAccount[accountId]) {
+		return Promise.resolve(usersInAccount[accountId].map((id) => ({ id, email: id })));
```

Comment:
this will be a ticking timebomb if you hack it this way lol

addUserToAccount takes in an email, so you should really be returning the email that was passed in originaly (maybe you need another dictionary of userIdToEmail, or change the usersInAccount object)